index.html

```diff
@@ -1,26 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Twitch Chat Counter</title>
   <link rel="stylesheet" href="https://ez-css.now.sh">
 </head>
+
 <body>
   <div>
     <center>
-      <marquee><p>Type 1 in the chat!</p></marquee>
+      <marquee>
+        <p>Type 1 in the chat!</p>
+      </marquee>
       <em>(broadcaster only)</em>
       <br />
       <code>!start-count | !end-count | !clear-count</code>
       <br />
       <br />
     </center>
     <small id="status"></small>
-    <h1 id="count">Waiting for count...</h1>
+    <h1> <span id="count">Waiting for count...</span> <span id="percent"></span></h1>
```

Comment:
To stick with what you've done above, maybe put the spam onto a new line? Cleanliness and consistency of code

Reply:
Actually, I personally think that it looks much better when both the count and percentage are on the same line instead of one below the other.